Bottom: f078fd059fc2978eed65db807d4ff3a85b141bf5
Top:    b412a60f93a59eef9701910985457882346fd56b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-05 18:00:55 +0100

Refresh of tag-the-from-vertex-with-the

---

diff --git a/src/polygon.c b/src/polygon.c
index ae0e65b..3ac0aa9 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -299,7 +299,7 @@ degree_circle (PLINE * c, Coord X, Coord Y /* <- Center */, Coord radius, Vector
     {
       /* NB: the caller added the first vertex, and will add the last vertex, hence the -1 */
       range = POLY_CIRC_SEGS * sweep / 360 - 1;
-      for (i = 0; i < range - 1; i++)
+      for (i = 0; i < range/* SHOULD NOT BE -1 */- 1; i++)
         {
           /* rotate the vector */
           t1 = rotate_circle_seg[0] * e1 + rotate_circle_seg[1] * e2;
